src/utils/CertificateProvider.js

```diff
@@ -599,6 +601,7 @@ export default class CertificateProvider {
       .then(_ =>
         openssl('req', {
           new: true,
+          CAserial: getFilePath('server.srl'),
```

Comment:
I think you're right that we just needed to take this arg out of the 'req' calls, and leave it in the 'x509' ones.
This one appears to be still left, but when I remove it, everything is fine - I don't think there were any differences in expected parameters between versions after all.

If you can confirm that with this change, it still works for you then we'll be good to go!
To do that, the easiest way is to wipe `~/.flipper`, which will force another cert exchange.

```suggestion
```

Reply:
Ah, great catch - left this in from the first time. Thanks!
Removing that doesn't change the behavior, seems to work expected 👍